azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -255,8 +255,9 @@ public String call() throws Exception {
           // max 5 secs
           String jsonString = refreshPair.getSecond().get(5, TimeUnit.SECONDS);
           executor.setExecutorInfo(ExecutorInfo.fromJSONString(jsonString));
-          logger.info("Successfully refreshed ExecutorInfo for executor: "
-            + executor);
+          logger.info(String.format(
```

Comment:
As discussed, it will have about 4 lines with 25-30 chars each. We will retaining it as info for the time being .